### ISSUE TITLE
【SF】【A11y】[SplitButton]: [Missing aria-label for arrow button in splitButton]

### ISSUE THREAD

**Issue by tandan0813** [SAP/ui5-webcomponents#9295]
### Describe the bug

When used SplitButton, there will have an A11y issue shows missing accessbile text in arrow icon button.
![image](https://github.com/SAP/ui5-webcomponents-react/assets/10113427/865e3dfc-e8da-44f2-bd98-83e03c432db1)

### Isolated Example

https://stackblitz.com/edit/github-nvtvkb?file=src%2FApp.tsx

### Reproduction steps

Refer isolated example.

### Expected Behaviour

Add aria-label "more" for arrow button in splitButton

### UI5 Web Components for React Version

1.29.2

### UI5 Web Components Version

1.24.4

### Browser

Chrome

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by kineticjs** [comment]
Hello @SAP/ui5-webcomponents-topic-b,

The issue is about missing aria lable for the menu button of the ui5-split-button.
In the screenreader spec I found the following:
"
- Add Add aria-haspopup=”menu” and aria-controls=”[IDREF_ASSOCIATED_MENU_OR_POPOVER]” to button that opens menu
- aria-labelledby for Label und tooltip
"
Also, the similar sap.m.Menu control in openui5 has an aria-label="Open Menu".
Can you check further on your side.
Thanks

**Comment by unazko** [comment]
Hello @tandan0813 and @SAP/ui5-webcomponents-topic-b,

The accessibility specification described how a menu button should behave.
The `ui5-split-button` is generally used as a menu button, but it could also be used for other purposes.

We'll need to provide an API like the following in order to provide flexibility:
https://sap.github.io/ui5-webcomponents/docs/advanced/accessibility/#accessibilityattributes

Best regards,
Boyan
